src/guide/resources/paloma-ledger.md

```diff
@@ -0,0 +1,200 @@
+# Ledger Nano Support for Paloma
+
+Using a hardware wallet to store your keys greatly improves the security of your crypto assets. The Ledger device acts as an enclave of the seed and private keys, and the process of signing transaction takes place within it. No private information ever leaves the Ledger device. The following is a short tutorial on using the Cosmos Ledger app with the Gaia CLI or the [Keplr](https://www.keplr.app/) wallet extension.
+
+At the core of a Ledger device there is a mnemonic seed phrase that is used to generate private keys. This phrase is generated when you initialize you Ledger. The mnemonic is compatible with Cosmos and can be used to seed new accounts.
+
+::: danger
+Do not lose or share your 24 words with anyone. To prevent theft or loss of funds, it is best to keep multiple copies of your mnemonic stored in safe, secure places. If someone is able to gain access to your mnemonic, they will fully control the accounts associated with them.
+:::
+
+## Install the Cosmos Ledger application
+
+Installing the `Cosmos` application on your ledger device is required before you can use either [Keplr](#keplr-+-ledger-nano) or [`palomad`](#gaia-cli-+-ledger-nano). To do so, you need to:
+
+1. Install [Ledger Live](https://shop.ledger.com/pages/ledger-live) on your desktop machine.
+<!-- markdown-link-check-disable -->
+2. Using Ledger Live, [update your Ledger Nano S with the latest firmware](https://support.ledger.com/hc/en-us/articles/360002731113?docs=true).
+<!-- markdown-link-check-enable -->
+3. On the Ledger Live application, navigate to the `Manager` menu .
+    ![manager](../images/ledger-tuto-manager.png)
+4. Connect your Ledger Nano device and allow Ledger Manager from it.
+5. On the Ledger Live application, Search for `Cosmos`.
+    ![search](../images/ledger-tuto-search.png)
```

Comment:
@taariq also this image?

Reply:
https://hub.cosmos.network/main/assets/img/ledger-tuto-search.bb886e41.png this is the image for this one.